.templates/zigbee2mqtt/service.yml

```diff
@@ -3,8 +3,12 @@ zigbee2mqtt:
   image: koenkk/zigbee2mqtt:latest
   environment:
     - TZ=${TZ:-Etc/UTC}
+    - ZIGBEE2MQTT_CONFIG_SERIAL_PORT=/dev/ttyACM0
+    - ZIGBEE2MQTT_CONFIG_SERIAL_ADAPTER=zstack
     - ZIGBEE2MQTT_CONFIG_MQTT_SERVER=mqtt://mosquitto:1883
```

Comment:
Why is this line removed?

Reply:
Err, which line? Nothing in the above context looks like it's removed. I only see two lines being added. If you meant 'ZIGBEE2MQTT_CONFIG_FRONTEND=true' being commented out, it's because that variable is incompatible with Zigbee2MQTT v2. If a user wants to run v1, he/she will have to enable that line to activate the front end.

Reply:
I thought it had the `ZIGBEE2MQTT_CONFIG_MQTT_SERVER` lined removed!